.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+Issue tracker is **ONLY** used for reporting bugs.
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
```

Comment:
Priority of the bug 
Suggestion to base this on the number of impacted users and the level of broken functionality 
eg. 
high number of impacted users (all) cannot use any part of the IF = URGENT drop everything, fix now. 
5% of users don't get output on one parameters = LOW

Reply:
In the open source working group repo, we have a couple of templates on this:

[ bug issue template](https://github.com/Green-Software-Foundation/opensource-wg/issues/new?assignees=&labels=&projects=&template=bug-report.md&title=)

[incident postmortem report ](https://github.com/Green-Software-Foundation/opensource-wg/issues/new?assignees=Sophietn&labels=&projects=&template=incident-postmortem-report.md&title=YYYY-MM-DD+Incident+%23)

We should agree a level of bug / incident severity that requires a postmortem